Podfile

```diff
@@ -8,5 +7,8 @@ target 'isic_kredit' do
   # Pods for isic_kredit
   pod 'SnapKit', '~> 4.2'
   pod 'Alamofire'
+  pod 'SwiftKeychainWrapper'
+  pod 'SwiftSoup'
+  pod 'JGProgressHUD'
```

Comment:
Nepoužívej pody bez určení verze. Vysvětloval jsem to na cviku, proč je to dobrý.

Reply:
Rad bych tam ty verze doplnil ale bohuzel nemuzu dohledat